src/components/Personalization/dom-actions/initDomActionsModules.js

```diff
@@ -47,21 +47,21 @@ export const DOM_ACTION_COLLECT_INTERACTIONS = "collectInteractions";
 
 export default () => {
   return {
-    [DOM_ACTION_SET_HTML]: createAction(setHtml),
-    [DOM_ACTION_CUSTOM_CODE]: createAction(prependHtml),
-    [DOM_ACTION_SET_TEXT]: createAction(setText),
-    [DOM_ACTION_SET_ATTRIBUTE]: createAction(setAttributes),
-    [DOM_ACTION_SET_IMAGE_SOURCE]: createAction(swapImage),
-    [DOM_ACTION_SET_STYLE]: createAction(setStyles),
-    [DOM_ACTION_MOVE]: createAction(move),
-    [DOM_ACTION_RESIZE]: createAction(resize),
-    [DOM_ACTION_REARRANGE]: createAction(rearrangeChildren),
-    [DOM_ACTION_REMOVE]: createAction(removeNode),
+    [DOM_ACTION_SET_HTML]: createAction(setHtml, true),
+    [DOM_ACTION_CUSTOM_CODE]: createAction(prependHtml, true),
```

Comment:
What is your thinking for allowing custom code to be re-run? We don't know if the custom code is idempotent or not.